src/core_plugins/kibana/public/management/sections/indices/index_pattern_list/components/list/list.js

```diff
@@ -35,27 +36,71 @@ export class List extends Component {
 
   renderList() {
     const { indexPatterns } = this.props;
-    return indexPatterns && indexPatterns.length ? (
-      <div>
-        {
-          indexPatterns.map(pattern => {
-            return (
-              <div key={pattern.id} >
-                <EuiButtonEmpty size="xs" href={pattern.url} data-test-subj="indexPatternLink">
-                  {pattern.default ? <Fragment><i aria-label="Default index pattern" className="fa fa-star" /> </Fragment> : ''}
-                  {pattern.active ? <strong>{pattern.title}</strong> : pattern.title} {pattern.tag ? (
-                    <Fragment key={pattern.tag.key}>
-                      {<EuiBadge color={pattern.tag.color || 'primary'}>{pattern.tag.name}</EuiBadge> }
-                    </Fragment>
-                  ) : null}
-                </EuiButtonEmpty>
-                <EuiSpacer size="xs"/>
-              </div>
-            );
-          })
-        }
-      </div>
-    ) : null;
+
+    if (indexPatterns && indexPatterns.length) {
+      return (
+        <div>
+          {
+            indexPatterns.map(pattern => {
+              const { id, default: isDefault, active, url, title, tag } = pattern;
+
+              let icon;
+
+              if (isDefault) {
+                icon = (
+                  <Fragment>
+                    <em
+                      aria-label={(
```

Comment:
`aria-label` expects a string, not a React Component.

![image](https://user-images.githubusercontent.com/11524637/47700512-cdd62980-dc27-11e8-932e-385c14a7228a.png)

Please wrap `List` class into `injectI18n` and translate `aria-label` using `props.intl.formatMessage` method:
`this.props.intl.formatMessage({ id: 'kbn.management.indexPatternList.defaultIndexPatternIconAriaLabel', defaultMessage: 'Default index pattern' }))`